package.json

```diff
@@ -18,7 +18,6 @@
     "bootstrap-no-build-no-concurrency": "lerna bootstrap --no-ci --force-local --ignore-scripts --concurrency 1",
     "bootstrap-no-build-quiet": "lerna bootstrap --loglevel=error --no-ci --force-local --ignore-scripts",
     "update": "lerna version --exact --force-publish --no-git-tag-version --no-push",
-    "publish": "lerna publish --skip-git --yes --exact --repo-version $VERSION",
```

Comment:
Are we removing it since it wasn't actually used in the Bamboo publish script?

Reply:
It was actually not just not used, but actively causing problems <facepalm emoji>